bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -169,18 +169,100 @@ private void processEvent(String content) {
+                } else {
+                    ipCameraHandler.logger.debug("Unrecognised Access control Dahua event, content={}", content);
+                }
+                break;
+            case "SIPRegisterResult":
+                if ("Pulse".equals(action)) {
+                    if (content.contains("\"Success\" : true"))
+                        ipCameraHandler.logger.debug("Camera register at SIP success");
+                } else {
+                    ipCameraHandler.logger.debug("Unrecognised SIP Dahua event, content={}", content);
+                }
+                break;
+            case "ProfileAlarmTransmit":
+                if ("Start".equals(action)) {
+                    if (content.contains("DoorMagnetism"))
+                        ipCameraHandler.logger.debug("Camera Magnetic lock fail warning");
+                }
+                if ("Stop".equals(action)) {
+                    if (content.contains("DoorMagnetism"))
+                        ipCameraHandler.logger.debug("Camera Magnetic lock fail warning gone");
+                } else {
+                    ipCameraHandler.logger.debug("Unrecognised Alarm Dahua event, content={}", content);
+                }
+                break;
+            case "DoorStatus":
+                if ("Pulse".equals(action)) {
+                    if (content.contains("\"Relay\" : true")) {
+                        ipCameraHandler.setChannelState(CHANNEL_DOOR_UNLOCK, OnOffType.OFF);
+                        ipCameraHandler.logger.debug("Dahua door lock unlocked");
+                    } else if (content.contains("\"Status\" : \"Close\"")) {
+                        ipCameraHandler.setChannelState(CHANNEL_DOOR_CONTACT, OnOffType.ON);
+                        ipCameraHandler.logger.debug("Dahua door contact closed");
+                    } else if (content.contains("\"Status\" : \"Open\"")) {
+                        ipCameraHandler.setChannelState(CHANNEL_DOOR_CONTACT, OnOffType.OFF);
+                        ipCameraHandler.logger.debug("Dahua door contact opened");
+                    } else {
+                        ipCameraHandler.logger.debug("Unrecognised Door status Dahua event, content={}", content);
+                    }
+                }
+                break;
             case "AlarmLocal":
                 if ("Start".equals(action)) {
                     if (content.contains("index=0")) {
                         ipCameraHandler.setChannelState(CHANNEL_EXTERNAL_ALARM_INPUT, OnOffType.ON);
+                    } else if (content.contains("index=3")) {
+                        ipCameraHandler.setChannelState(CHANNEL_EXIT_BUTTON, OnOffType.ON);
+                        ipCameraHandler.logger.debug("Dahua exit button pressed");
                     } else {
                         ipCameraHandler.setChannelState(CHANNEL_EXTERNAL_ALARM_INPUT2, OnOffType.ON);
+                        ipCameraHandler.logger.debug("External alarm Dahua event, content={}", content);
```

Comment:
Remove the logging IMHO I do not see the content as being helpful for a user and the event.log already has the channel changing state logged.

Reply:
As far as I remember this one could be rather interesting - there could be more indexes for different cameras/doorphones for different situations, so maybe this should be kept, just with loglevel trace maybe.

Reply:
If it could be useful then happy to keep this one.